nodeshot/ui/default/static/ui/nodeshot/js/views/search.js

```diff
@@ -27,6 +27,7 @@
 
         events:{
             'keyup input': 'keyupOnSearch',
+            'click #general-search-input-button': 'ClickOnSearch',
```

Comment:
`click @ui.icon` should work, see the `ui`section to understand: https://github.com/mapeveri/nodeshot/blob/master/nodeshot/ui/default/static/ui/nodeshot/js/views/search.js#L24

`ClickOnSearch`should be `clickOnSearch`(lowercase first letter)